server/__init__.py

```diff
@@ -65,6 +65,8 @@ def create_app():
         cfg = import_string('configmodule.StagingConfig')()
     elif os.environ.get('FLASK_ENV') == 'development':
         cfg = import_string('configmodule.DevelopmentConfig')()
+    elif os.environ.get('FLASK_ENV') == 'kubernetes':
```

Comment:
should we call this minikube / kubernetes-staging

Reply:
Yeah. This is to make it easy for testing now. I will update it when it's ready to migrate over